CHANGELOG.md

```diff
@@ -5,6 +5,121 @@ information on the list of deprecated flags and APIs please have a look at
+* Password stdin [docker/cli#271](https://github.com/docker/cli/pull/271)
+* Don't attempt to remove unsupported resources on older daemon [docker/cli#277](https://github.com/docker/cli/pull/277)
+* Return an empty stats if "container not found" [moby/moby#34004](https://github.com/moby/moby/pull/34004)
++ Add cluster events details in swagger.yml [moby/moby#34035](https://github.com/moby/moby/pull/34035)
+* Matching events using ID and name [moby/moby#33377](https://github.com/moby/moby/pull/33377)
+- Fix stack output stream [docker/cli#298](https://github.com/docker/cli/pull/298)
+* Progress: Show task error in place of progress bar [docker/cli#259](https://github.com/docker/cli/pull/259)
+* Login: print a big warning when using --password [docker/cli#270](https://github.com/docker/cli/pull/270)
++ Add `--detach` to docker scale [docker/cli#243](https://github.com/docker/cli/pull/243)
+* Set ping version even on error [moby/moby#33827](https://github.com/moby/moby/pull/33827)
+* Progress: Show success message when finished counting down [docker/cli#260](https://github.com/docker/cli/pull/260)
+* Hide `--detach` for docker < 17.05 [docker/cli#219](https://github.com/docker/cli/pull/219)
+* Progress: Show progress of replicated tasks before they are assigned [docker/cli#237](https://github.com/docker/cli/pull/237)
+* Improve volume spec parsing [docker/cli#152](https://github.com/docker/cli/pull/152)
+* Allow Proxy Configuration in config.json [docker/cli#93](https://github.com/docker/cli/pull/93)
++ Add `scope` filter in `GET /networks/(id or name)` [moby/moby#33630](https://github.com/moby/moby/pull/33630)
+* Enable client side digest pinning for stack deploy [docker/cli#121](https://github.com/docker/cli/pull/121)
+* Allow --detach and --quiet flags when using --rollback [docker/cli#144](https://github.com/docker/cli/pull/144)
+
+### Builder
+
++ Add DOCKER_BUILD_ARGS to support build with proxy [docker/cli#335](https://github.com/docker/cli/pull/335)
+- Fix multiple `COPY --from` in multistage builds [moby/moby#34063](https://github.com/moby/moby/pull/34063)
++ Add path filtering to build session client [moby/moby#33859](https://github.com/moby/moby/pull/33859)
+- Fix handling of remote "git@" notation [moby/moby#33696](https://github.com/moby/moby/pull/33696)
+* Incrementally sending build context [docker/cli#231](https://github.com/docker/cli/pull/231)
+* Warn on empty continuation lines [moby/moby#33719](https://github.com/moby/moby/pull/33719)
+* Implement long running interactive session and sending build context incrementally [moby/moby#32677](https://github.com/moby/moby/pull/32677)
+- Fix copy —from conflict with force pull [moby/moby#33735](https://github.com/moby/moby/pull/33735)
+* Move file coping from the daemon to the builder [moby/moby#33454](https://github.com/moby/moby/pull/33454)
+- Fix behavior of absolute paths in .dockerignore [moby/moby#32088](https://github.com/moby/moby/pull/32088)
```

Comment:
Fixed a bug where .dockerignore file entries with a leading `/` would not match anything